operator/pkg/reconciliation/defaults.go

```diff
@@ -0,0 +1,11 @@
+package reconciliation
+
+var (
+	// Provides reasonable defaults for the logger container.
+	DefaultsLoggerContainer = buildResourceRequirements(500, 256)
+
+	// Provides reasonable defaults for the configuration container.
+	DefaultsConfigInitContainer = buildResourceRequirements(1000, 256)
```

Comment:
Nothing hugely CPU bound here, although we should test, too. Too high of a CPU request will make issues for our test and examples running on small setups, though.

```suggestion
	DefaultsConfigInitContainer = buildResourceRequirements(250, 256)
```